src/pages/Homepage.svelte

```diff
@@ -79,28 +79,19 @@
   <h1 class="sr-only">{$t('content.index.title')}</h1>
   <div class="mb-8">
     <div class="flex flex-col items-center justify-center m-10 gap-5">
-      <!-- svelte-ignore a11y-no-noninteractive-element-interactions -->
-      <img
-        src={introVideoPlaceholderImage}
```

Comment:
Please can we remove the image import and the image file?

Reply:
https://github.com/microbit-foundation/ml-trainer/pull/208/commits/daf2d1639cef80be155a9f0e2be48821d6f07df9